examples/seismic/acoustic/operators.py

```diff
@@ -25,14 +30,22 @@ def laplacian(field, m, s, kernel):
 def iso_stencil(field, m, s, damp, kernel, **kwargs):
     """
     Stencil for the acoustic isotropic wave-equation:
-    u.dt2 - H + damp*u.dt = 0
-    :param field: Symbolic TimeFunction object, solution to be computed
-    :param m: square slowness
-    :param s: symbol for the time-step
-    :param damp: ABC dampening field (Function)
-    :param kwargs: forwad/backward wave equation (sign of u.dt will change accordingly
-    as well as the updated time-step (u.forwad or u.backward)
-    :return: Stencil for the wave-equation
+    u.dt2 - H + damp*u.dt = 0.
+
+    Parameters
+    ----------
+    field : TimeFunction
+        The computed solution.
+    m : Function or float
+        Square slowness.
+    s : float or Scalar
+        The time dimension spacing.
+    damp : Function
+        The damping field for absorbing boundary condition.
+    forward : boolean
```

Comment:
nitpicking: the actual type is ``bool``, not ``boolean``. Could be changed here and everywhere (I spotted other occurrences)

Reply:
Three (3) boolean changed to bool.